plugin/v2/settable.go

```diff
@@ -0,0 +1,89 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
+	value string
+}
+
+var (
+	allowedSettableFieldsEnv     = []string{"value"}
+	allowedSettableFieldsArgs    = []string{"value"}
+	allowedSettableFieldsDevices = []string{"path"}
+	allowedSettableFieldsMounts  = []string{"source"}
+)
+
+func newSettables(args []string) []settable {
+	settables := []settable{}
+	for _, arg := range args {
+		settables = append(settables, newSettable(arg))
+	}
+	return settables
+}
+
+func newSettable(arg string) settable {
+	var set settable
+	if i := strings.Index(arg, "="); i < 0 {
+		set.name = arg
+		set.value = "true"
```

Comment:
I thought about that too. The only problem I could see is that it's not intuitive, because most programs will consider `""` to mean false. But maybe it's not an issue for now.